applications/subswap/contract/scripts/simpletokendeploy.js

```diff
@@ -5,15 +5,24 @@
 // will compile your contracts, add the Hardhat Runtime Environment's members to the
 // global scope, and execute the script.
 const hre = require("hardhat");
+const deploy = require("../deploy.config.json");
 
 async function main() {
```

Comment:
just an improvement. I think we shall have try catch with some nice error message in case of failure